docker/pypi/wmagent/run.sh

```diff
@@ -607,30 +490,29 @@ main(){
     check_wmasecrets
     check_docker_init || {
         (deploy_to_host)         || { err=$?; echo "ERROR: deploy_to_host"; exit $err ;}
-        (deploy_to_container)    || { err=$?; echo "ERROR: deploy_to_container"; exit $err ;}
-        (activate_agent)         || { err=$?; echo "ERROR: activate_agent"; exit $err ;}
-        start_services
-        sleep 5
         (check_databases)        || { err=$?; echo "ERROR: check_databases"; exit $err ;}
+        (activate_agent)         || { err=$?; echo "ERROR: activate_agent"; exit $err ;}
         (init_agent)             || { err=$?; echo "ERROR: init_agent"; exit $err ;}
-        sleep 5
         (agent_tweakconfig)      || { err=$?; echo "ERROR: agent_tweakconfig"; exit $err ;}
         (agent_resource_control) || { err=$?; echo "ERROR: agent_resource_control"; exit $err ;}
         (agent_upload_config)    || { err=$?; echo "ERROR: agent_upload_config"; exit $err ;}
+        echo $WMA_BUILD_ID > $wmaInitUsing
         (check_docker_init)      || { err=$?; echo "ERROR: DockerBuild vs. HostConfiguration version missmatch"; exit $err ; }
-
         echo && echo "Docker container has been initialised! However you still need to:"
-        echo "  1) Double check agent configuration: less current/config/wmagent/config.py"
-        echo "  2) Start the agent with on of the bellow commands: "
-        echo "      manage start-agent     (from inside a running wmagent container)"
-        echo "      ./wmagent-docker-run & (from the host)"
+        echo "  1) Double check agent configuration: less /data/[dockerMount]/srv/wmagent/current/config/config.py"
+        echo "  2) Start the agent with one of the following commands: "
+        echo "     * From inside a running wmagent container:"
+        echo "       docker exec -it wmagent bash"
```

Comment:
This message isn't clear. I think the best would be to say something like:
```
* Access the running WMAgent container:
  docker exec -it wmagent bash
* From inside the container:
  manage start-agent
```

Reply:
done